[0.476s] Invoking command in '/workspaces/codespace/ros2/project/tutorial/src/examples/rclpy/topics/pointcloud_publisher': AMENT_PREFIX_PATH=/workspaces/codespace/ros2/project/tutorial/install/examples_rclpy_pointcloud_publisher PYTEST_ADDOPTS=--tb=short --junit-xml=/workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher/pytest.xml --junit-prefix=examples_rclpy_pointcloud_publisher -o cache_dir=/workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher/.pytest_cache PYTHONPATH=/workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher:/workspaces/codespace/ros2/project/tutorial/install/examples_rclpy_pointcloud_publisher/lib/python3.10/site-packages /usr/bin/python3 -m pytest
[0.623s] [1m============================= test session starts ==============================[0m
[0.623s] platform linux -- Python 3.10.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.0
[0.623s] cachedir: /workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher/.pytest_cache
[0.623s] rootdir: /workspaces/codespace/ros2/project/tutorial/src/examples/rclpy, configfile: pytest.ini
[0.623s] plugins: colcon-core-0.15.2
[0.634s] [1mcollecting ... [0m[1m
[0.634s] collected 0 items / 3 errors                                                   [0m
[0.659s] 
[0.659s] ==================================== ERRORS ====================================
[0.659s] [31m[1m_____ ERROR collecting topics/pointcloud_publisher/test/test_copyright.py ______[0m
[0.659s] [31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/rclpy/topics/pointcloud_publisher/test/test_copyright.py'.
[0.659s] Hint: make sure your test modules/packages have valid Python names.
[0.659s] Traceback:
[0.659s] /usr/lib/python3.10/importlib/__init__.py:126: in import_module
[0.659s]     return _bootstrap._gcd_import(name[level:], package, level)
[0.659s] test/test_copyright.py:15: in <module>
[0.659s]     from ament_copyright.main import main
[0.660s] E   ModuleNotFoundError: No module named 'ament_copyright'[0m
[0.660s] [31m[1m_______ ERROR collecting topics/pointcloud_publisher/test/test_flake8.py _______[0m
[0.660s] [31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/rclpy/topics/pointcloud_publisher/test/test_flake8.py'.
[0.660s] Hint: make sure your test modules/packages have valid Python names.
[0.660s] Traceback:
[0.660s] test/test_flake8.py:18: in <module>
[0.660s]     from ament_flake8.main import main
[0.660s] E   ModuleNotFoundError: No module named 'ament_flake8'
[0.660s] 
[0.660s] During handling of the above exception, another exception occurred:
[0.660s] /usr/lib/python3.10/importlib/__init__.py:126: in import_module
[0.660s]     return _bootstrap._gcd_import(name[level:], package, level)
[0.660s] test/test_flake8.py:20: in <module>
[0.660s]     from ament_flake8.main import main_with_errors
[0.660s] E   ModuleNotFoundError: No module named 'ament_flake8'[0m
[0.660s] [31m[1m_______ ERROR collecting topics/pointcloud_publisher/test/test_pep257.py _______[0m
[0.660s] [31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/rclpy/topics/pointcloud_publisher/test/test_pep257.py'.
[0.660s] Hint: make sure your test modules/packages have valid Python names.
[0.660s] Traceback:
[0.660s] /usr/lib/python3.10/importlib/__init__.py:126: in import_module
[0.661s]     return _bootstrap._gcd_import(name[level:], package, level)
[0.661s] test/test_pep257.py:15: in <module>
[0.661s]     from ament_pep257.main import main
[0.661s] E   ModuleNotFoundError: No module named 'ament_pep257'[0m
[0.661s] - generated xml file: /workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher/pytest.xml -
[0.661s] =========================== short test summary info ============================
[0.661s] ERROR test/test_copyright.py
[0.661s] ERROR test/test_flake8.py
[0.661s] ERROR test/test_pep257.py
[0.661s] !!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!
[0.661s] [31m============================== [31m[1m3 errors[0m[31m in 0.06s[0m[31m ===============================[0m
[0.675s] Invoked command in '/workspaces/codespace/ros2/project/tutorial/src/examples/rclpy/topics/pointcloud_publisher' returned '2': AMENT_PREFIX_PATH=/workspaces/codespace/ros2/project/tutorial/install/examples_rclpy_pointcloud_publisher PYTEST_ADDOPTS=--tb=short --junit-xml=/workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher/pytest.xml --junit-prefix=examples_rclpy_pointcloud_publisher -o cache_dir=/workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher/.pytest_cache PYTHONPATH=/workspaces/codespace/ros2/project/tutorial/build/examples_rclpy_pointcloud_publisher:/workspaces/codespace/ros2/project/tutorial/install/examples_rclpy_pointcloud_publisher/lib/python3.10/site-packages /usr/bin/python3 -m pytest
